android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/SpeakerViewHolder.java

```diff
@@ -50,9 +55,17 @@ public SpeakerViewHolder(View itemView, Context context) {
         //Attach onClickListener for ViewHolder
         itemView.setOnClickListener(view -> {
             String speakerName = speaker.getName();
+            
+            Bundle bundle = null;
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
+                bundle = ActivityOptions.makeSceneTransitionAnimation((Activity) this.context,speakerImage,speakerImage.getTransitionName()).toBundle();
```

Comment:
In this case, your cast of context to Activity worked, but in future after any refactoring, it may happen so that the passed context is no longer Activity and may lead to a `ClassCastException`

Please change the solution to something more robust :wink: